docs/install.md

```diff
@@ -20,6 +24,7 @@ spec:
   logLevel: debug
   targetNamespace: hive
 ```
+  * apply the yaml file doing `$ oc apply -f hive_config.yaml`
```

Comment:
Wrap command in ```, indentation may be incorrect.

```suggestion
  * Apply hive_config.yaml,
      ```
      oc apply -f hive_config.yaml
      ```
```

Reply:
Done. Thanks!